publisher-sdk/src/androidTest/java/com/criteo/publisher/integration/DfpHeaderBiddingFunctionalTest.java

```diff
@@ -672,11 +691,34 @@ public void onAdLoaded() {
       }
 
       @Override
-      public void onAdFailedToLoad(int i) {
+      public void onAdFailedToLoad(LoadAdError loadAdError) {
+        LoggerFactory.getLogger(DfpHeaderBiddingFunctionalTest.class)
+            .debug("onAdFailedToLoad for reason %d. "
+                    + "See: https://developers.google.com/android/reference/com/google/android/gms/ads/doubleclick/PublisherAdRequest",
+                loadAdError);
+        isLoaded.countDown();
+      }
+    }
+
+    public class SyncAdManagerInterstitialAdLoadCallback extends AdManagerInterstitialAdLoadCallback {
+      private AdManagerInterstitialAd _adManagerInterstitialAd;
+
+      public AdManagerInterstitialAd getAdManagerInterstitialAd() {
```

Comment:
Nit: in a simple callback in the test codebase, I wouldn't mind if you omit the getter :)

Reply:
done